AndroidBound/src/main/java/solutions/alterego/androidbound/android/converters/BooleanToVisibilityConverter.java

```diff
@@ -13,15 +13,16 @@
 
     public static final String PARAMETER_INVERT_TAG = "invert";
 
-    public static String getConverterName() {
+    @Override
+    public String getBindingName() {
         return CONVERTER_NAME;
     }
 
     @Override
     public Object convert(Object value, Class<?> targetType, Object parameter, Locale culture) {
         boolean result = false;
 
-        if (value instanceof Boolean) {
+        if (value != null && value instanceof Boolean) {
```

Comment:
instanceof checks against null as well `null instanceof Boolean` is always evaluated false